pkgs/applications/misc/keeweb/default.nix

```diff
@@ -0,0 +1,63 @@
+{ stdenv, fetchurl, appimageTools, undmg }:
+let
+  inherit (stdenv.hostPlatform) system;
+  throwSystem = throw "Unsupported system: ${system}";
+
+  pname = "keeweb";
+  version = "1.15.6";
+  name = "${pname}-${version}";
+
+  suffix = {
+    x86_64-linux = "linux.AppImage";
+    x86_64-darwin = "mac.dmg";
+  }.${system} or throwSystem;
+
+  src = fetchurl {
+    url = "https://github.com/keeweb/keeweb/releases/download/v${version}/KeeWeb-${version}.${suffix}";
+    sha256 = {
+      x86_64-linux = "1jz61k745dygr1c0imv411xq4i4hajqiph97bmm78v1x02753ama";
+      x86_64-darwin = "029zmh83mrf9qqk5yjplshv3nlba71l4h21xywmp6dkb4vr676xr";
+    }.${system} or throwSystem;
+  };
+
+  appimageContents = appimageTools.extractType2 {
+    inherit name src;
+  };
+
+  meta = with stdenv.lib; {
+    description = "Free cross-platform password manager compatible with KeePass";
+    homepage = "https://keeweb.info/";
+    license = licenses.mit;
+    maintainers = with maintainers; [ sikmir ];
+    platforms = [ "x86_64-linux" "x86_64-darwin" ];
+  };
+
+  linux = appimageTools.wrapType2 rec {
+    inherit name src meta;
```

Comment:
I'm not familiar with appimageTools, but suspect this needs a pname too.

Reply:
`appimageTools.wrapType2` and `appimageTools.extractType2` require `name` argument. See https://github.com/NixOS/nixpkgs/blob/be2c00811b3fbb42b321ae497b9870b068fd4a79/pkgs/build-support/appimage/default.nix#L50 and https://github.com/NixOS/nixpkgs/blob/be2c00811b3fbb42b321ae497b9870b068fd4a79/pkgs/build-support/appimage/default.nix#L29